backend/core/src/shared/email/email.service.ts

```diff
@@ -148,6 +148,60 @@ export class EmailService {
     )
   }
 
+  public async newlisting(listing: Listing, user: User) {
+    await this.loadTranslationsForUser(user)
+    if (this.configService.get<string>("NODE_ENV") == "production") {
+      Logger.log(
+        `Preparing to send a listing email to ${user.email} from ${this.configService.get<string>(
+          "EMAIL_FROM_ADDRESS"
+        )}...`
+      )
+    }
+
+    const rentRange = await this.getRentRange(listing)
+    await this.send(
+      user.email,
+      "New Listing",
```

Comment:
Should we get this from polyglot instead?

Reply:
Yeah, but this should also be in the translation PR right?